core/proto/keytransparency_v1_types/keytransparency_v1_types.proto

```diff
@@ -236,3 +236,19 @@ message GetMutationsResponse {
   // results.
   string next_page_token = 7;
 }
+
+// GetMonitoringRequest contains the input parameters of the GetMonitoring APIs.
+message GetMonitoringRequest {
+  // start specifies the start epoch number from which monitoring results will
+  // be returned (ranging from [start, latestObserved] and starting at 1).
+  int64 start = 1;
+}
+
+message GetMonitoringResponse {
+  // smr contains the map root for the sparse Merkle Tree signed with the
```

Comment:
For convenience: a pass / fail field. 

Proof fields for each type of failure.